packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -75,26 +75,36 @@ class DebuggerController extends DisposableController
 
   VmServiceWrapper _lastService;
 
-  void _handleConnectionAvailable(VmServiceWrapper service) {
+  void _handleConnectionAvailable(VmServiceWrapper service) async {
```

Comment:
Should we update this to return `Future<void>`

Reply:
done